pkg/api/customization/cluster/formatter.go

```diff
@@ -34,6 +34,10 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 		resource.AddAction(request, "enableMonitoring")
 	}
 
+	if _, ok := resource.Values["rancherKubernetesEngineConfig"]; ok {
+		resource.AddAction(request, "backupEtcd")
```

Comment:
could we add this here since it already exists https://github.com/rancher/rancher/blob/486b190eb52ae8b5b7ca1cec840fab39b44202c8/pkg/api/customization/cluster/formatter.go#L28